recipe/meta.yaml

```
@@ -56,6 +57,7 @@ outputs:
       host:
         - nccl         # [linux and cuda_compiler != "None"]
       run:
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.15") }}  # [osx and x86_64]
         - __cuda                   # [cuda_compiler != "None"]
         {% if (cuda_compiler_version or "").startswith("11") %}
         - cuda-version >={{ cuda_compiler_version }},<12
```

Think we may also want to update this to allow CUDA 11.2 (if we are reusing CUDA 11.8 builds on older CUDA versions)

```suggestion
        - cuda-version >=11.2,<12
```

---

Should we allow CUDA 11.2 at all?

---

At present I would like to make sure we are supporting users of CUDA 11.2+ somehow. If reusing CUDA 11.8 builds on 11.2+ does that, then would help simplify things here